File: test/fixtures/output/java/unirest/full.java
HttpResponse<JsonNode> jsonResponse = Unirest.post("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value")
			.header("Accept", "application/json")
			.header("Content-Type", "application/x-www-form-urlencoded")
			.header("Cookie", "foo=bar; bar=baz")
			.body("foo=bar")
			.asJson();

File: test/fixtures/output/node/unirest/full.js
var unirest = require("unirest");

var req = unirest("POST", "http://mockbin.com/har");

var CookieJar = unirest.jar();
CookieJar.add("foo=bar","http://mockbin.com/har");
CookieJar.add("bar=baz","http://mockbin.com/har");
req.jar(CookieJar);

req.query({
  "foo": [
    "bar",
    "baz"
  ],
  "baz": "abc",
  "key": "value"
});

req.headers({
  "accept": "application/json",
  "content-type": "application/x-www-form-urlencoded"
});

req.form({
  "foo": "bar"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

